pkg/crypto/aes_test.go

```diff
@@ -0,0 +1,61 @@
+package crypto
+
+import (
+	"encoding/base64"
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func Test_General(t *testing.T) {
+	req := require.New(t)
+	// ensure that it works the first time
+	req.NoError(NewAESCipher())
+	cipherKey := ToString()
+
+	// ensure that values that are encrypted can be decrypted again
+	testValue := "this is a test value"
+	testEncrypted := Encrypt([]byte(testValue))
+	testDecrypted, err := Decrypt(testEncrypted)
+	req.NoError(err)
+	req.NotEqual(testValue, string(testEncrypted))
+	req.Equal(testValue, string(testDecrypted))
+
+	// ensure that rerunning it does not cause an error or change the cipherKey
+	req.NoError(NewAESCipher())
+	newCipherKey := ToString()
+	req.Equal(cipherKey, newCipherKey)
+
+	testDecrypted, err = Decrypt(testEncrypted)
+	req.NoError(err)
+	req.Equal(testValue, string(testDecrypted))
+
+	// ensure that string values of the cipherKey can be added
+	err = InitFromString(cipherKey)
+	req.NoError(err)
+
+	fmt.Printf("cipherKey: %s encrypted: %s\n", cipherKey, base64.StdEncoding.EncodeToString(testEncrypted))
```

Comment:
this need?

Reply:
🤦‍♂️ no, that was for me to get some values to hardcode below